api/occupancy/OccupancyOperations.php

```diff
@@ -17,6 +17,28 @@ class OccupancyOperations
     const MEDIUM = 'https://api.irail.be/terms/medium';
     const HIGH = 'https://api.irail.be/terms/high';
 
+    public static function getOccupancyURI($vehicle, $from, $date) {
+        $occupancyDeparture = self::getOccupancyTrip($vehicle, $from, $date);
+
+        if(!is_null($occupancyDeparture)) {
+            return self::NumberToURI($occupancyDeparture->occupancy);
+        } else {
+            return self::getUnknown();
+        }
+    }
+
+    private static function getOccupancyTrip($vehicle, $from, $date) {
+        $dotenv = new Dotenv(dirname(dirname(__DIR__)));
+        $dotenv->load();
+        $mongodb_url = getenv('MONGODB_URL');
+        $mongodb_db = getenv('MONGODB_DB');
```

Comment:
Should we disable occupancy when these variables were not set or when we were unable to connect?